src/api/app/policies/report_policy.rb

```diff
@@ -32,8 +32,7 @@ def create?
 
   def notify?
     return false unless Flipper.enabled?(:content_moderation, user)
-    return true if User.moderators.blank? && (user.admin? || user.staff?)
-    return true if user.moderator?
+    return true if user.moderator? || user.admin? || user.staff?
 
     false
```

Comment:
```suggestion
    user.moderator? || user.admin? || user.staff?
```

Nitpick: I find the `return true if ...` and then the next line `false` more convoluted...

Reply:
Let me change it!